### ISSUE TITLE
JSX: Nested String Interpolation

### ISSUE THREAD

**Issue by jaredmoody** [coffeelint/coffeelint#27]
This error gets thrown all over the place for interpolation inside any jsx.

For example, here's the first component example from the [react tutorial](https://reactjs.org/tutorial/tutorial.html#overview) in coffeescript:

```
class ShoppingList extends React.Component
  render: =>
    <div className="shopping-list">
      <h1>Shopping List for {@props.name}</h1>
      <ul>
        <li>Instagram</li>
        <li>WhatsApp</li>
        <li>Oculus</li>
      </ul>
    </div>
```

```
yarn run v1.21.1
$ /Users/jaredmoody/code/coffeelint_test/node_modules/.bin/coffeelint test.jsx.coffee
  ⚡ test.jsx.coffee
     ⚡ #4: Nested string interpolation is forbidden.
     ⚡ #5: Nested string interpolation is forbidden.
```

**Comment by UziTech** [comment]
Is this still an issue in v3.1.2?

**Comment by UziTech** [comment]
This seems to be fixed in v3.1.2
